src/components/apiComponents/languageSelector.jsx

```
@@ -15,27 +15,37 @@ const languages = [
+          return (
+            <button
+              key={language.name}
+              className={`-ms-px inline-flex min-w-16 flex-col items-center justify-center gap-1 border border-gray-200 px-10 py-2 text-xs font-medium text-gray-800 shadow-sm transition-colors first:ms-0 first:rounded-s-lg last:rounded-e-lg hover:bg-gray-50 focus:z-10 focus:bg-accent-300 focus:outline-none focus:ring-1 disabled:pointer-events-none disabled:opacity-50 dark:border-neutral-700 dark:bg-neutral-900 dark:text-white dark:hover:bg-neutral-800 ${
+                activeLanguage === language.name
+                  ? 'bg-accent-300 text-black hover:bg-accent-300 dark:bg-accent-300 dark:text-black dark:hover:bg-accent-300'
+                  : ''
+              } ${isUmiDisabled ? 'opacity-50 cursor-not-allowed' : ''}`}
+              onClick={() => !isUmiDisabled && setActiveLanguage(language.name)}
+              disabled={isUmiDisabled}
+              title={isUmiDisabled ? 'UMI support coming soon' : ''}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)